generators/app/index.js

```diff
@@ -13,6 +14,8 @@ module.exports = class extends Generator {
     const defaultAuthor = `${gitName} <${gitEmail}>`
 
     const defaultPackageName = path.basename(this.destinationRoot())
+    const convertToVariableName = pipe(words, map(upperFirst), join(''))
```

Comment:
Instead of `convertToVariableName`, how about something that more clearly describes the function's purpose, such as `convertToUmdGlobalName`?